dff/script/__init__.py

```diff
@@ -15,10 +15,8 @@
 from .core.normalization import (
     normalize_label,
     normalize_condition,
-    normalize_transitions,
     normalize_response,
     normalize_processing,
-    normalize_script,
 )
```

Comment:
Not related to the PR, but:

Why import these functions if they are not supposed to be used by the user?

Reply:
https://trello.com/c/vKJsysZq